### PR TITLE
(partial resolve for #94) Added search fields to admin's teachers view, added more fields to…

### PR DESCRIPTION
**Pull request by Aszanat**
… show on teachers list (still in admin's view).

### CODE REVIEW

File: oioioi/teachers/models.py [hunk 1]
```diff
@@ -24,6 +24,7 @@ class Teacher(models.Model):
     )
     is_active = models.BooleanField(default=False, verbose_name=_("active"))
     school = models.CharField(max_length=255, verbose_name=_("school"))
+    join_date = models.DateField(auto_now_add=True)
```
Comment (twalen) [inline oioioi/teachers/models.py:27]: migration is needed!

**Review by twalen — CHANGES_REQUESTED** [review]
Please add a migration for new model field

**Comment by A-dead-pixel** [comment]
I think you added ``extra/easy-install/.venv`` and ``oioioi.log`` by mistake to the last commit @Aszanat. Patterns like ``**/.venv*/`` and ``**/oioioi.log`` (or maybe ``**/*.log``?) should probably be added to ``.gitignore``.

File: oioioi/teachers/admin.py [hunk 2]
```diff
@@ -38,6 +40,31 @@ def formfield_for_foreignkey(self, db_field, request, **kwargs):
...
+
+    def teacher_login(self, instance):
+        if not instance.user:
+            return ''
+        return instance.user.get_username()
+
+    teacher_first_name.short_description = _("First name")
+    teacher_last_name.short_description = _("Last name")
+    teacher_email.short_description = _("Email address")
+    teacher_login.short_description = _("Userame")
```
Comment (twalen) [inline oioioi/teachers/admin.py:66]: Userame -> Username

**Review by twalen — CHANGES_REQUESTED** [review]
Looks good. Please fix typo and we can merge it.

**Pull request merged**